example/gardener-local/kind/cluster-ha-skaffold.yaml

```diff
@@ -0,0 +1,176 @@
+apiVersion: kind.x-k8s.io/v1alpha4
```

Comment:
To follow the existing structure, this file should probably be put into `example/gardener-local/kind-ha/cluster-skaffold.yaml`.
Also, can you provide a `cluster.yaml` for the local case?